recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,3 @@
+# To bootstrap. Should be removed when these entries go into the aggregate cbc.yaml.
```

Comment:
This file can be removed completely. This was needed before we had _any_ `cuda_compiler_version` entry in the cbc.yaml. compiler version 12.4 to build this will be fine IIRC

Reply:
(having this I think will cause issues when you remove your `channel` config from the abs.yaml and merge the PR)

Reply:
Thanks. I removed it.